cpl_replacement_cache.go

```diff
@@ -1,97 +0,0 @@
-package kbucket
-
-import (
```

Comment:
I really don't think we need all this anymore as it needlessly adds complexity and makes the code/RT state hard to reason about.

The `fixLowPeers` call  in the DHT along with shorter intervals for refreshes(10 minutes now) should keep the RT in a good state.